### DISCUSSION TITLE
Using NFC Card for paying an invoice presented in the POS app

### DISCUSSION THREAD

**Discussion by rgaitan1982** [btcpayserver/btcpayserver#6522] [Technical Support]
Hi:

I am the webmaster of https://bitcoinpaybox.es/ third party host for local in-store merchants.

I have reviewed this guide https://docs.btcpayserver.org/Conference-PoS-guide/ but I would like to know if it would work using on-chain instead of LN. Setting up On-Chain Payout Processor instead of Lightning Payout Processor.

I mean, can I program a blank NFC card (NTAG424DNA) for charging it with 0.001 BTC for instance and use it for paying to bc1q addresses instead of ln addresses?

![image](https://github.com/user-attachments/assets/b1b78ac9-4cf2-4003-8a4b-df9eba82146b)

My merchants do not use LN yet, they are confortable receiving just BTC on-chain payments for now.

Thanks!

**Comment by pavlenex** [comment]
Not possible on-chain, only with LN.
